pkg/yamlutil/parser.go

```diff
@@ -174,3 +176,28 @@ func (p *Parser) unmarshal(reader io.Reader) (*parsed, error) {
 func (p *Parser) buildConfigFromParsed(parsed *parsed, b Builder) error {
 	return b.BuildFromParsed(parsed.objects)
 }
+
+func ParseMultiYamlFile(fileName string) ([][]byte, error) {
```

Comment:
Can we accept an `io.Reader`? It makes the function compatible with non-file sources and simplifies testing as files aren't necessary.

The function can also be simplified to `SplitDocuments` particularly considering we aren't really parsing the file.

```go
func SplitDocuments(r io.Reader) ([][]byte, error) { ... }
```

```go
func main() {
	reader := ...
	yaml.SplitDocuments(reader)
}
```